cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -45,10 +45,12 @@ def normalize_paths(paths)
       (is_extension && path.match('/'))
       raise ArgumentError.new("Invalid path: #{path}")
     end
+    single_path = !path.match('\*')
```

Comment:
I don't understand why the query regex is appended in every path that contains '*' and never otherwise.  

Can you explain the logic here, perhaps in a comment?  

Is the the right thing to do in a pattern like "*.jpg"?